docs/source/cmake/cmake_packages_without_sudo.rst

```diff
@@ -0,0 +1,95 @@
+
+
+Install a CMake package without sudo privileges
+===============================================
+
+To install a CMake package or library without sudo privileges, we need to define a directory to which we have access. For instance, 
+in :code:`~/`.
+
+
+Create a custom folder
+----------------------
+
+In this tutorial, we are going to create a custom folder :code:`~/opt` containing 
+the folders :code:`lib` and :code:`include`. This will be our directory to install all our CMake packages.
+
+
+.. code-block:: console
+
+    cd ~/
+    mkdir opt && cd opt
```

Comment:
some of these might be reused so if you'd prefer you can do
`mkdir -p opt`
and so on.

With this option, `mkdir` will not complain if the folder already exists.